src/Sylius/Bundle/CoreBundle/Migrations/Version20201208105207.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+final class Version20201208105207 extends AbstractMigration
+{
+    /** @var ContainerInterface */
+    private $container;
+
+    public function getDescription(): string
+    {
+        return 'Set details and shipment_id on shipping adjustments.';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->setDefaultAdjustmentData();
+
+        $adjustments = $this->getShipmentAdjustmentsWithData();
+
+        foreach ($adjustments as $adjustment) {
+            $this->updateAdjustment(
+                (int) $adjustment['id'],
+                (int) $adjustment['shipping_id'],
+                json_encode(['shippingMethodCode' => $adjustment['shipment_code'], 'shippingMethodName' => $adjustment['label']])
```

Comment:
There is missing information about tax rates, which is very important for details of tax adjustment